builders/build/substrate-api/chopsticks.md

```diff
@@ -0,0 +1,167 @@
+---
+title: How to use Chopsticks to Fork Moonbeam
+description: Follow this tutorial to learn the basic of how to use Chopstick to locally fork the entirety of a Moonbeam network.
+---
+
+# How to Use Chopsticks to Fork Moonbeam
+
+![Chopsticks diagram](/images/builders/build/substrate-api/chopsticks/chopsticks-banner.png)
+
+## Introduction {: #introduction }
+
+Chopsticks is a library for building Substrate-based blockchain applications. Most importantly, Chopsticks provides a developer-friendly method of locally forking existing Substrate based chains. This allows developers to test and experiment with their own custom blockchain configurations in a local development environment, without the need to deploy a live network.  
+
+Overall, Chopsticks aims to simplify the process of building blockchain applications on Substrate and make it accessible to a wider range of developers.
+
+## Checking Prerequeisites {: #checking-prerequisites }
+
+To use Chopsticks, you will need the following:  
+
+- Have a recent version of [Rust installed](https://www.rust-lang.org/tools/install){target=_blank} 
+- Have [Yarn](https://classic.yarnpkg.com/en/){target=_blank} installed  
+- Have [Docker](https://docs.docker.com/get-docker/){target=_blank} installed
+
+## Configuring Chopsticks {: configuring-chopsticks }
+
+First, install chopsticks from its GitHub repository, add dependencies, and build it:  
+
+```
+git clone --recurse-submodules https://github.com/AcalaNetwork/chopsticks.git && \
+cd chopsticks && \
+yarn && \
+yarn build-wasm
+```
+
+Chopsticks includes a set of [YAML](https://yaml.org/){target=_blank} configuration files that can be used to create a local copy of a variety of Substrate chains. You can view each of the default configuration files within the `configs` folder. Moonbeam, Moonriver, and Moonbase Alpha all have default files available. The example configuration below is the current  configuration for Moonbeam:  
+
+```yaml
+endpoint: wss://wss.api.moonbeam.network
+mock-signature-host: true
+db: ./db.sqlite
+
+import-storage:
+  System:
+    Account:
+      -
+        -
```

Comment:
just curious, do you know why these are necessary?

Reply:
No, but they appear for all of the other networks as well. I would guess that it would have to do with how the accounts storage works internally but that's just speculation